configs/filter.yaml

```diff
@@ -1,3 +1,7 @@
+#tenants:
+#    tenant_id:
+#        - <tenant_id1>
+#        - <tenant_id2>
```

Comment:
I guess we should write somewhere, that with this patch we can filter only by 1 tenant... And I guess with those one, that has been specified first. Isn't is?